index.org

```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
    * ~units~ Blocks that create units;
    * ~upgrade~ Things that upgrade the player such as mech pads;
    * ~effect~ Things for storage or passive effects.
-** TODO Zone
+** Zone
+    You can add your mod's campaign map by this
+    
+    | field              | type            | default    |
+    |--------------------+-----------------+------------|
+    | generator          | [[Generator]]   |            |
+    | baseLaunchCost     | [[ItemStack]]   |            |
+    | launchCost         | [[ItemStack]]   |            |
+    | startingItems      | [[ItemStack]]   |            |
+    | conditionWave      | int             | MAX_VALVE  |
+    | alwaysUnlocked     | boolean         |            |
+    | launchPeriod       | int             | 10         |
+    | loadout            | [[Loadout]]     | basicShard |
+    | resources          | [[Item]]        |            |
+    | rules              | [[Rules]]       |            |
+    | requirements       | [[Objective]]   |            |
+    | configureObjective | [[Objective]]   |            |
+    | defaultStartingItems | [[ItemStack]] |            |
+    
+*** Loadout
+    Loadout is the blueprint when your core landed.
+    
+    In Loadout:
+    | '>' is stands for conveyor face to right |
```

Comment:
I'm having a hard time understand what this table is supposed to tell the user:

* give it a header and columns.
* the wording `x is stands for y` doesn't really make sense.
* `'>'` should be in a code block, not in single quotes.

Despite that I'm not sure I understand exactly what it's saying, I'll have to take a look at the source. I know you're defining characters to be used in relation to a block, but I'm not entirely sure why you're doing it.